app/src/main/assets/carapplications/ChassisKodes.kt

```diff
@@ -0,0 +1,102 @@
+enum class ChassisCode(val brand: String, val chassis: String) {
+    // Mini codes
+    R50("MINI", "Cooper"),
+    R53("MINI", "Cooper"),
+    R56("MINI", "Cooper"),
+    F55("MINI", "Cooper"),
+    F56("MINI", "Cooper"),
+    R52("MINI", "Convertible"),
+    R57("MINI", "Convertible"),
+    F57("MINI", "Convertible"),
+    R55("MINI", "Clubman"),
+    F54("MINI", "Clubman"),
+    R58("MINI", "Coupe"),
+    R59("MINI", "Roadster"),
+    R61("MINI", "Paceman"),
+    R60("MINI", "Countryman"),
+    F60("MINI", "Countryman"),
+
+    // BMW Codes
+    F48("BMW", "X1"),
+    F25("BMW", "X3"),
+    E84("BMW", "X1"),
+    E83("BMW", "X3"),
+    E72("BMW", "X6"),
+    E71("BMW", "X6"),
+    E70("BMW", "X5"),
+    E53("BMW", "X5"),
+    E26("BMW", "M1"),
+    E86("BMW", "Z4 Coupe"),
+    E89("BMW", "Z4 Roadster"),
+    E85("BMW", "Z4 Roadster"),
+    E25("BMW", "Concept Turbo"),
+
+    E9("BMW",	"2 Series"),
```

Comment:
These rows have slightly different spacing hehe

Reply:
An artefact of the gimmicks I did to convert between formats 😁